qiskit/providers/ibmq/api/session.py

```diff
@@ -199,9 +199,26 @@ def request(  # type: ignore[override]
 
             if self.access_token:
                 message = message.replace(self.access_token, '...')
-                # Replace the original message on the `RequestException` as well.
+                # Replace the original message on the `RequestException`.
                 ex.args = (message,)
```

Comment:
You don't need this line anymore, right? Since `_modify_chained_exception_messages` will replace the message?

Reply:
Yeah, you're right! We wouldn't need that line after all. Thanks @jyu00!